Lib/concurrent/futures/process.py

```diff
@@ -578,7 +612,7 @@ class BrokenProcessPool(_base.BrokenExecutor):
 
 class ProcessPoolExecutor(_base.Executor):
     def __init__(self, max_workers=None, mp_context=None,
-                 initializer=None, initargs=()):
+                 initializer=None, initargs=(), maxtasksperchild=None):
```

Comment:
1) Perhaps make this a keyword-only argument?
2) Change the name to `max_tasks_per_child` (for PEP8 compliance)

Reply:
I agree with both these points.

Reply:
I only named it `maxtasksperchild` because I was trying to match the original multiprocessing library.